playhouse/postgres_ext.py

```diff
@@ -330,6 +343,18 @@ def get_parse_map(self):
             json_path=self._parse_json_path)
         return parse_map
 
+    def generate_update(self, query):
+        sql, params = super(PostgresqlExtCompiler, self).generate_update(query)
+        if getattr(query, '_returning', None):
+            clauses = [SQL(' RETURNING')]
+            returning_clause = Clause(*query._returning)
+            returning_clause.glue = ', '
+            clauses.append(returning_clause)
+            returning_sql, _ = self.build_query(clauses)
+            sql += returning_sql
```

Comment:
Note that you are not appending any parameterized values from the `RETURNING` clause to the list of values from the main query.